cupy/sorting/count.py

```diff
@@ -1,18 +1,25 @@
 from cupy import core
 
 
-def count_nonzero(x):
+def count_nonzero(x, axis=None):
     """Counts the number of non-zero values in the array.
 
     Args:
         x (cupy.ndarray): The array for which to count non-zeros.
-
+        axis (int or tuple): optionalAxis or tuple of axes along which to count
+            non-zeros. Default is None, meaning that non-zeros will be counted
+            along a flattened version of ``a``
```

Comment:
Sorry I forget that. I will rename the identifier `x` to `a` because `numpy` renamed it from v.1.12.0 with the `axis` option.